config/mongo.go

```diff
@@ -57,3 +63,43 @@ func LoadMongoDBFromEnv() *MongoDB {
 	}
 	return &mongo
 }
+
+func (m *MongoDB) mustMode(s *mgo.Session) {
+	if m.Mode == "" {
```

Comment:
Probably no need for this if statement as the `default` case of the switch will take care of this scenario.